automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/FireCloudView.scala

```diff
@@ -1,6 +1,6 @@
 package org.broadinstitute.dsde.firecloud.page
 
-import org.broadinstitute.dsde.firecloud.test.WebBrowserUtil
```

Comment:
Future work, outside the scope of this PR: remove `org.broadinstitute.dsde.firecloud` and import `org.broadinstitute.dsde.workbench` from workbench-libs

Reply:
That would be https://broadinstitute.atlassian.net/browse/DSDEQA-160 / https://broadinstitute.atlassian.net/browse/GAWB-2801